services/profile-service/src/app.ts

```diff
@@ -0,0 +1,21 @@
+import bodyParser from "body-parser";
+import cors from "cors";
+import express, { Express, NextFunction, Request, Response } from "express";
+import "reflect-metadata";
+import userRoutes from "./router/user-routes";
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
+
+app.use("/user", userRoutes);
```

Comment:
Might need input from @Acerizm , to use plural nouns `/users`?

Reply:
sure plural nouns would be better

Reply:
I have changed it to plural